dna/utils.py

```diff
@@ -1,4 +1,37 @@
 import numpy as np
+import pandas as pd
+from matplotlib import pyplot as plt
+import os
 
 def rank(values):
     return np.argsort(values)[::-1]
+
+
+def plot_spearman(ranked_data: pd.DataFrame, actual_data: pd.DataFrame, name: str, directory: str, score: float):
```

Comment:
Let's put as many scores in on the same plot as possible, similar to how it is in the [link](https://github.com/byu-dml/d3m-dynamic-neural-architecture/issues/37#issuecomment-501427904), instead of having one plot per score. This allows us to compare all the scores together.

Reply:
Are you saying you want to combine spearman and pearson scores into the same plot?

Reply:
Yes, I want them in the same plot. All scores are computed using the same predictions and targets. Make the code dynamic so that it includes all the scores that are provided.